### ISSUE TITLE
2.5.4 broking on server side rendering

### ISSUE THREAD

**Issue by odelucca** [react-ga/react-ga#315]
Expected behaviour
---
When importing the package it would need to wait the initialization of GoogleAnalytics.initialize() until running any code.

Current behaviour
---
After upgrading to the new version (from 2.5.3 to 2.5.4) when I run:
```
import GoogleAnalytics from 'react-ga';
```

It automatically calls a function. The problem is, my app is Server Side Rendered, so I just need to run it on the front-end, and, because of that, I got the following error:
```
ReferenceError: window is not defined
    at Object.<anonymous> (/home/odelucca/Servers/yurah/web/site-ssr/node_modules/react-ga/dist/react-ga.js:10:4)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.react-ga (/home/odelucca/Servers/yurah/web/site-ssr/src/.next/server/static/development/pages/home.js:4814:18)
    at __webpack_require__ (/home/odelucca/Servers/yurah/web/site-ssr/src/.next/server/static/development/pages/home.js:23:31)
    at Module../modules/core/actions.js (/home/odelucca/Servers/yurah/web/site-ssr/src/.next/server/static/development/pages/home.js:1946:66)
    at __webpack_require__ (/home/odelucca/Servers/yurah/web/site-ssr/src/.next/server/static/development/pages/home.js:23:31)
    at Module../modules/chat/components/Conclusao.js (/home/odelucca/Servers/yurah/web/site-ssr/src/.next/server/static/development/pages/home.js:1192:71)
    at __webpack_require__ (/home/odelucca/Servers/yurah/web/site-ssr/src/.next/server/static/development/pages/home.js:23:31)
    at Module../modules/chat/components/Chat.js (/home/odelucca/Servers/yurah/web/site-ssr/src/.next/server/static/development/pages/home.js:1033:68)
    at __webpack_require__ (/home/odelucca/Servers/yurah/web/site-ssr/src/.next/server/static/development/pages/home.js:23:31)
    at Module../pages/home.js (/home/odelucca/Servers/yurah/web/site-ssr/src/.next/server/static/development/pages/home.js:4497:87)
    at __webpack_require__ (/home/odelucca/Servers/yurah/web/site-ssr/src/.next/server/static/development/pages/home.js:23:31)
    at Object.3 (/home/odelucca/Servers/yurah/web/site-ssr/src/.next/server/static/development/pages/home.js:4637:18)
    at __webpack_require__ (/home/odelucca/Servers/yurah/web/site-ssr/src/.next/server/static/development/pages/home.js:23:31)
    at module.exports.../node_modules/react-phone-number-input/style.css (/home/odelucca/Servers/yurah/web/site-ssr/src/.next/server/static/development/pages/home.js:91:18)
    at Object.<anonymous> (/home/odelucca/Servers/yurah/web/site-ssr/src/.next/server/static/development/pages/home.js:94:10)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at _callee$ (/home/odelucca/Servers/yurah/web/site-ssr/node_modules/next/dist/server/require.js:85:46)
    at tryCatch (/home/odelucca/Servers/yurah/web/site-ssr/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/home/odelucca/Servers/yurah/web/site-ssr/node_modules/regenerator-runtime/runtime.js:288:22)
    at Generator.prototype.(anonymous function) [as next] (/home/odelucca/Servers/yurah/web/site-ssr/node_modules/regenerator-runtime/runtime.js:114:21)
    at asyncGeneratorStep (/home/odelucca/Servers/yurah/web/site-ssr/node_modules/@babel/runtime-corejs2/helpers/asyncToGenerator.js:5:24)
    at _next (/home/odelucca/Servers/yurah/web/site-ssr/node_modules/@babel/runtime-corejs2/helpers/asyncToGenerator.js:27:9)
    at /home/odelucca/Servers/yurah/web/site-ssr/node_modules/@babel/runtime-corejs2/helpers/asyncToGenerator.js:34:7
    at new Promise (<anonymous>)
    at new F (/home/odelucca/Servers/yurah/web/site-ssr/node_modules/core-js/library/modules/_export.js:36:28)
    at /home/odelucca/Servers/yurah/web/site-ssr/node_modules/@babel/runtime-corejs2/helpers/asyncToGenerator.js:23:12
    at requirePage (/home/odelucca/Servers/yurah/web/site-ssr/node_modules/next/dist/server/require.js:69:23)
    at _callee3$ (/home/odelucca/Servers/yurah/web/site-ssr/node_modules/next/dist/server/render.js:223:196)
    at tryCatch (/home/odelucca/Servers/yurah/web/site-ssr/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/home/odelucca/Servers/yurah/web/site-ssr/node_modules/regenerator-runtime/runtime.js:288:22)
    at Generator.prototype.(anonymous function) [as next] (/home/odelucca/Servers/yurah/web/site-ssr/node_modules/regenerator-runtime/runtime.js:114:21)
    at asyncGeneratorStep (/home/odelucca/Servers/yurah/web/site-ssr/node_modules/@babel/runtime-corejs2/helpers/asyncToGenerator.js:5:24)
    at _next (/home/odelucca/Servers/yurah/web/site-ssr/node_modules/@babel/runtime-corejs2/helpers/asyncToGenerator.js:27:9)
```

I tried a lot of different combinations, and the only thing that stopped this errors was reming the import of react-ga.

I think that it is because it is trying to get the window in my server side.

How to fix it
---
I just went back to the 2.5.3 version. Can you guys fix this in the new version?

**Comment by anajavi** [comment]
Can you please try the fix in #314 ? It should fix the problem, but it is not confirmed.

**Comment by SimeonC** [comment]
This should be fixed in the new 2.5.5 version.
